tls/s2n_handshake_type.h

```diff
@@ -18,74 +18,74 @@
 
-#define IS_FULL_HANDSHAKE(conn)             \
-    ( s2n_handshake_type_check_flag(conn, FULL_HANDSHAKE) )
+#define IS_FULL_HANDSHAKE(conn) \
+    (s2n_handshake_type_check_flag(conn, FULL_HANDSHAKE))
 
-#define IS_RESUMPTION_HANDSHAKE(conn)       \
-    ( !IS_FULL_HANDSHAKE(conn) && IS_NEGOTIATED(conn) )
+#define IS_RESUMPTION_HANDSHAKE(conn) \
+    (!IS_FULL_HANDSHAKE(conn) && IS_NEGOTIATED(conn))
 
-#define IS_CLIENT_AUTH_HANDSHAKE(conn)      \
-    ( s2n_handshake_type_check_flag(conn, CLIENT_AUTH) )
+#define IS_CLIENT_AUTH_HANDSHAKE(conn) \
+    (s2n_handshake_type_check_flag(conn, CLIENT_AUTH))
 
-#define IS_CLIENT_AUTH_NO_CERT(conn)        \
-    ( IS_CLIENT_AUTH_HANDSHAKE(conn) && s2n_handshake_type_check_flag(conn, NO_CLIENT_CERT) )
+#define IS_CLIENT_AUTH_NO_CERT(conn) \
+    (IS_CLIENT_AUTH_HANDSHAKE(conn) && s2n_handshake_type_check_flag(conn, NO_CLIENT_CERT))
 
 #define IS_TLS12_PERFECT_FORWARD_SECRECY_HANDSHAKE(conn) \
-    ( s2n_handshake_type_check_tls12_flag(conn, TLS12_PERFECT_FORWARD_SECRECY) )
+    (s2n_handshake_type_check_tls12_flag(conn, TLS12_PERFECT_FORWARD_SECRECY))
 
-#define IS_OCSP_STAPLED(conn)               \
-    ( s2n_handshake_type_check_tls12_flag(conn, OCSP_STATUS) )
+#define IS_OCSP_STAPLED(conn) \
+    (s2n_handshake_type_check_tls12_flag(conn, OCSP_STATUS))
 
 #define IS_ISSUING_NEW_SESSION_TICKET(conn) \
-    ( s2n_handshake_type_check_tls12_flag(conn, WITH_SESSION_TICKET) )
+    (s2n_handshake_type_check_tls12_flag(conn, WITH_SESSION_TICKET))
 
-#define IS_NPN_HANDSHAKE(conn)               \
-    ( s2n_handshake_type_check_tls12_flag(conn, WITH_NPN) )
+#define IS_NPN_HANDSHAKE(conn) \
+    (s2n_handshake_type_check_tls12_flag(conn, WITH_NPN))
 
-#define IS_HELLO_RETRY_HANDSHAKE(conn)      \
-    ( s2n_handshake_type_check_tls13_flag(conn, HELLO_RETRY_REQUEST) )
+#define IS_HELLO_RETRY_HANDSHAKE(conn) \
+    (s2n_handshake_type_check_tls13_flag(conn, HELLO_RETRY_REQUEST))
 
-#define IS_MIDDLEBOX_COMPAT_MODE(conn)      \
-    ( s2n_handshake_type_check_tls13_flag(conn, MIDDLEBOX_COMPAT) )
+#define IS_MIDDLEBOX_COMPAT_MODE(conn) \
+    (s2n_handshake_type_check_tls13_flag(conn, MIDDLEBOX_COMPAT))
 
-#define WITH_EARLY_DATA(conn)               \
-    ( s2n_handshake_type_check_tls13_flag(conn, WITH_EARLY_DATA) )
+#define WITH_EARLY_DATA(conn) \
+    (s2n_handshake_type_check_tls13_flag(conn, WITH_EARLY_DATA))
 
-#define WITH_EARLY_CLIENT_CCS(conn)         \
-    ( s2n_handshake_type_check_tls13_flag(conn, EARLY_CLIENT_CCS) )
+#define WITH_EARLY_CLIENT_CCS(conn) \
+    (s2n_handshake_type_check_tls13_flag(conn, EARLY_CLIENT_CCS))
 
 typedef enum {
-    INITIAL                         = 0,
-    NEGOTIATED                      = 1,
-    FULL_HANDSHAKE                  = 2,
-    CLIENT_AUTH                     = 4,
-    NO_CLIENT_CERT                  = 8,
+    INITIAL = 0,
+    NEGOTIATED = 1,
+    FULL_HANDSHAKE = 2,
+    CLIENT_AUTH = 4,
+    NO_CLIENT_CERT = 8,
```

Comment:
It would be nice if you could get it to align these values.

Reply:
I can do that with clang-format off/on. but it doesn't feel worth it. 

Unfortunately the alignment here is controlled by AlignConsecutiveAssignments